### ISSUE TITLE
Multicall support

### ISSUE THREAD

**Issue by tomvlk** [MiniGod/node-gbxremote#4]
Would be great to have some kind of multicall support. Currently it doesn't right?

**Comment by MiniGod** [comment]
You are correct, it does not, but I don't see any reason for them unless you want to do atomic operations with multiple methods.
Can you give an example case for this?

**Comment by tomvlk** [comment]
See some code from here: https://github.com/ManiaJS/ManiaJS/blob/master/src/server/client.js#L136
Can be really easy done with mutlicall, (or with simulating multicall maybe, just executing in series and return back).

**Comment by MiniGod** [comment]
In that case you can use [`Promise.all`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all).

**Comment by tomvlk** [comment]
multicall isn't more efficient for network? well, for the performance (and hangs) you don't have to do in JS.
